data-processing-lib/doc/overview.md

```diff
@@ -45,6 +45,7 @@ To learn more consider the following:
 * [Transform Exceptions](transform-exceptions.md)
 * [Transform Runtimes](transform-runtimes.md)
 * [Transform Examples](transform-tutorial-examples.md)
+* [Data Access Factory S3 Example](using-data-access-factory.md)
```

Comment:
I think a more general discussion of the DataAccessFactory concept and what it facilitates would be a good starting point - a big picture sort of overview, including input selection, checkpointing, reading and writing, etc.
Then a discussion on an S3 and/or local example.